tfbpshiny/tabs/individual_regulator_compare_module.py

```diff
@@ -378,44 +405,76 @@ def _():
     @reactive.calc
     def has_column_changes():
         """Reactive to check if there are changes in column selection."""
-        current_replicate_selection_table_selection = set(
-            input.replicate_selection_table_columns.get() or []
+        current_replicate_selection_general_qc_table_selection = set(
+            input.replicate_selection_table_general_qc_columns.get() or []
+        )
+        current_replicate_selection_insert_table_selection = set(
+            input.replicate_selection_table_insert_table_columns.get() or []
         )
-        confirmed_replicate_selection_table_selection = set(
+        confirmed_replicate_selection_general_qc_table_selection = set(
             selected_replicate_selection_table_columns.get()
         )
-
+        confirmed_replicate_selection_insert_table_selection = set(
+            selected_replicate_selection_table_insert_columns.get()
+        )
         current_summarized_binding_perturbation_selection = set(
             input.summarized_binding_perturbation_columns.get() or []
         )
         confirmed_summarized_binding_perturbation_selection = set(
             selected_summarized_binding_perturbation_columns.get()
         )
+        current_database_identifier_selection = set(
+            input.database_identifier_columns.get() or []
+        )
+        confirmed_database_identifier_selection = set(
+            selected_database_identifier_columns.get()
+        )
 
         return (
             current_summarized_binding_perturbation_selection
```

Comment:
not a huge fan of this -- would be nice of there was something more concise and maintainable